src/App.tsx

```diff
@@ -16,60 +15,125 @@ const goodsFromServer = [
 ];
 
 enum SortType {
-  NONE,
-  ALPABET,
-  LENGTH,
+  none,
+  alphabet,
+  length,
```

Comment:
Let's use ALLCAPS, cause it's mainly used for such constants values like this
```suggestion
  NONE,
  ALPABET,
  LENGTH,
```